README.md

```diff
@@ -1 +1,5 @@
-# gneiss
\ No newline at end of file
+# gneiss
+
+Canonically pronouced *nice*
+
+A compositional statistics and visualization toolbox
```

Comment:
The balance trees is a visualization.
It needs to be a part of this repo
On Jun 30, 2016 6:08 AM, "Antonio Gonzalez" <notifications@github.com>
wrote:

> In README.md
> <https://github.com/biocore/gneiss/pull/2#discussion_r69128056>:
>
> > \ No newline at end of file
> > +# gneiss
> > +
> > +Canonically pronouced *nice*
> > +
> > +A compositional statistics and visualization toolbox
>
> What kind of visualizations are you planning to add? Should they be their
> own repo or part of another one?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/biocore/gneiss/pull/2/files/d9d026e68a7b2594ef2b01bb9689afe9b2f0e1dc#r69128056>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AD_a3cs_j_2U_Pniz9RDn7sIcRO_nxu5ks5qQ7_RgaJpZM4JBs7r>
> .
>